README.md

```diff
@@ -11,41 +11,67 @@ using [Syft](https://github.com/anchore/syft).
 
 By default, this action will execute a Syft scan in the workspace directory
 and upload a workflow artifact SBOM in SPDX format. It will also detect
-if being run during a `release` and attach the SBOM
-as a release asset.
+if being run during a [GitHub release](https://docs.github.com/en/repositories/releasing-projects-on-github/about-releases)
+and upload the SBOM as a release asset.
 
 ## Example Usage
 
-### Scan a docker image
+### Scan a container image
 
-Use the `image` parameter
+To scan a container image from the Docker daemon, use the `image` parameter:
 
 ```yaml
 - uses: anchore/sbom-action@main
   with:
     image: example/image_name
 ```
 
+To use a specific container registry, specify a typical _registry path_ to the
+image including the registry host. This will
+attempt to use the Docker daemon and by association any previously configured
+credentials. If the Docker daemon is not available, a direct connection to the
+registry will be attempted:
+
+```yaml
+- uses: anchore/sbom-action@main
+  with:
+    image: ghcr.io/example/image_name:tag
+```
+
+To use an authenticated direct connection to the registry regardless of the
+Docker daemon availability, add credentials using
+`registry-username` and `registry-password`:
+
+```yaml
+- uses: anchore/sbom-action@main
+  with:
+    image: my-registry.com/my/image
+    registry-username: mr_awesome
+    registry-password: ${{ secrets.REGISTRY_PASSWORD }}
+```
```

Comment:
I think this is on the right track. I think the net result of today's decisions mean this can actually be simplified even further. Here's one suggestion:

```suggestion
To scan a container image, use the `image` parameter:

```yaml
- uses: anchore/sbom-action@main
  with:
    image: ghcr.io/example/image_name:tag
```

If the Docker daemon is available, the action will retrieve the image using the daemon. This allows any authentication available to the daemon to be used if needed when fetching the image.

If the Docker daemon is not available. the action will retrieve the image directly from the container registry.

You can also specify registry credentials explicitly, which will cause the action to retrieve the image from the registry directly (ignoring the Docker daemon altogether). To do this, specify the`registry-username` and `registry-password` parameters:

```yaml
- uses: anchore/sbom-action@main
  with:
    image: my-registry.com/my/image
    registry-username: mr_awesome
    registry-password: ${{ secrets.REGISTRY_PASSWORD }}
```
```

Reply:
This got a little messed up because GitHub's suggestion feature created a situation of "backticks within backticks" — but maybe the idea still got through 😄

Reply:
Good suggestion! It's been reworded a bit and @luhring approved! :)